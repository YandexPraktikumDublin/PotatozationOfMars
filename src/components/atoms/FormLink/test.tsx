import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { FormLink } from '.'

describe('<FormLink />', () => {
  it('should renders correct <FormLink />', () => {
    const wrapper = shallow(<FormLink text="test" path="test" />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
