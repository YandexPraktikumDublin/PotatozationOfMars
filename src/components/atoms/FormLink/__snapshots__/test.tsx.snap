// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormLink /> should renders correct <FormLink /> 1`] = `
<Link
  className="underline"
  to="test"
>
  test
</Link>
`;
